package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "build-main": "cross-env NODE_ENV=production webpack --config webpack.main.prod.config.js",
     "build-renderer": "cross-env NODE_ENV=production webpack --config webpack.renderer.prod.config.js",
-    "build": "npm run build-main && npm run build-renderer",
+    "build": "npm run build-main && npm run build-renderer && build-storybook",
```

Comment:
I think it would be better to put build-storybook in "prestorybook" task instead of prolonging build time here

Reply:
what would `prestorybook` be then? Tbh, I saw this from other repos. If this is used in CI, then better to remove.

Reply:
it's gonna execute everytime you call `yarn run storybook`

Reply:
That build is not required for development. I don't think we even need build-storybook (will we display somewhere those static files)?

Reply:
Nah, but default build will compile those files because they are in src